lib/optimizely/config_manager/async_scheduler.rb

```diff
@@ -75,10 +75,10 @@ def execution_wrapper(callback)
       loop do
         begin
           callback.call
-        rescue
+        rescue StandardError => e
           @logger.log(
             Logger::ERROR,
-            'Something went wrong when running passed function.'
+            "Something went wrong when executing passed callback. #{e.message}"
```

Comment:
Shouldn't this be sent to the error handler so that users can handle these errors as desired? (In particular, without invoking the error handler, users can't record these exceptions to their error reporting services.)